analyzer/src/test/kotlin/PackageManagerTest.kt

```diff
@@ -39,25 +39,26 @@ class PackageManagerTest : WordSpec({
             // entry for each package manager.
             result.keys shouldBe PackageManager.ALL.toSet()
 
-            result[Bundler] shouldBe listOf(File(projectDir, "Gemfile"))
-            result[GoDep] shouldBe listOf(File(projectDir, "Gopkg.toml"))
-            result[Gradle] shouldBe listOf(File(projectDir, "build.gradle"))
-            result[Maven] shouldBe listOf(File(projectDir, "pom.xml"))
-            result[NPM] shouldBe listOf(File(projectDir, "package.json"))
-            result[PhpComposer] shouldBe listOf(File(projectDir, "composer.json"))
-            result[PIP] shouldBe listOf(File(projectDir, "setup.py"))
-            result[SBT] shouldBe listOf(File(projectDir, "build.sbt"))
-            result[Stack] shouldBe listOf(File(projectDir, "stack.yaml"))
-            result[Yarn] shouldBe listOf(File(projectDir, "yarn.lock"))
+            result[Bundler.Factory()] shouldBe listOf(File(projectDir, "Gemfile"))
```

Comment:
There are no `hashCode` and `equals` implementations in PackageManagerFactory, so this is bound to fail, right?

Reply:
Right, on it.